README.md

```diff
@@ -45,25 +44,88 @@ Create an input configuration object and add a redux-inputs reducer to your redu
         inputs: createInputsReducer(inputConfig)
     });
 
-Note that `inputs` is the default mount point for redux-inputs state. You can
- configure this in your `inputConfig` object.
+### Step 4 - Connect
 
-    inputConfig = {
-        _form: {
-            reduxMountPoint: 'loginModal.userInputs'
-        }
-        ... other inputs config ...
-    }
+    import { connectWithInputs } from 'redux-inputs';
+    
+    YourForm = connectWithInputs(inputConfig)()(YourForm);
+    
+Connect your form with `connectWithInputs`, which takes the `inputConfig` and creates a function that has the same 
+interface as react-redux's `connect`. This will pass down these additional props to your component, in addition to any 
+other redux state specified:
 
-In this example your state would look like this:
+#### Props
 
-    {
-        loginModal: {
-            userInputs: { ... }
-        }
+- `inputProps` *{Object}* contains props for each input in the inputConfig
+
+- `inputActions` *{Object}*
```

Comment:
Rather than always passing these down as props, should we do something similar to `mapStateToProps` from redux? `connectWithInputs` would take `inputConfig` as well as a `mapInputActionsToProps` function? This would keep props clean on the component -- you'd only map those you needed.